napari/_qt/qt_event_loop.py

```diff
@@ -90,8 +142,7 @@ def gui_qt(*, startup_logo=False, gui_exceptions=False, force=False):
 
     app = get_app()
     if startup_logo and app.applicationName() == 'napari':
-        logopath = join(dirname(__file__), '..', 'resources', 'logo.png')
-        pm = QPixmap(logopath).scaled(
+        pm = QPixmap(NAPARI_ICON_PATH).scaled(
```

Comment:
Can the icon path here somehow get read from the `app`? I guess not really needed as you're calling `get_app` with defaults, but it just seems safer to only have this set once - i.e. maybe we store it as `app._icon_path`. Up to you, it's just a rough idea

Reply:
I actually looked into that a bit.  I was unable to find a way to extract the fileName from a [`QIcon`](https://doc.qt.io/qt-5/qicon.html), and I'd rather not set attributes on the application object (we do it with `_existed` but I want to get rid of that)...  In any case, I think the splash window and the application window icon are different enough that we don't necessarily always want one to be the other.

Reply:
yeah, ok, that's fine too!